.gitignore

```diff
@@ -2,3 +2,18 @@
 .DS_Store
```

Comment:
This line should go away.  This is user environment and doesn't belong in the project.